x-pack/plugins/whitelabelling/server/plugin.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CoreSetup, CoreStart, Plugin, PluginInitializerContext } from '@kbn/core/server';
+import { WhitelabellingRequestHandlerContext } from './types';
```

Comment:
NIT: import type (multiple occurrences across new files in the `whitelabelling` plugin)